### DISCUSSION TITLE
Adding curators to studio

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7159] [Help]
Do I remember well that there was an addon that allowed to add all your followers as curators of one of your studios? I cannot find it anymore (if any)

**Comment by BroJac5246** [comment]
This would be very prone to spam and would not make a good feature IMO.

**Comment by CST1229** [comment]
I'm fairly sure there wasn't ever any addon that did that. (Though there is one which lets you invite individual followers, `studio-followers`.)

**Reply by sfederici** [reply]
Yes, exactly, this one (to @BroJac5246: sorry, I didn't mean enroll without their consent)

**Reply by sfederici** [reply]
Thanks a lot
